### PR TITLE
Phase2-hgx318 Add a scenario 2026D94 using HGCal version V17 and HFNose

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add a scenario 2026D94 using HGCal version V17 and HFNose

#### PR validation:

Use theTheMatrix test workflow 41834.0 and overlap checking tool

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by srimanob** [comment]
test parameters:
- workflow = 41834.0

**Comment by srimanob** [comment]
@cmsbuild please test

**Comment by civanch** [comment]
+1

File: Geometry/ForwardCommonData/python/hfnoseV15ParametersInitialization_cfi.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+from Geometry.HGCalCommonData.hgcalEEV15ParametersInitialization_cfi import *
+
+hfnoseParametersInitialize = hgcalEEParametersInitialize.clone(
+    name  = cms.string("HGCalHFNoseSensitive"),
+    name2 = cms.string("HFNoseEE"),
+    nameW = cms.string("HFNoseWafer"),
+    nameC = cms.string("HFNoseCell"),
+    nameT = cms.string("HFNose"),
+    nameX = cms.string("HGCalHFNoseSensitive"),
```
Comment (perrotta) [inline Geometry/ForwardCommonData/python/hfnoseV15ParametersInitialization_cfi.py:11]:
```suggestion
    name  = "HGCalHFNoseSensitive",
    name2 = "HFNoseEE",
    nameW = "HFNoseWafer",
    nameC = "HFNoseCell",
    nameT = "HFNose",
    nameX = "HGCalHFNoseSensitive"
```
(safer syntax, as already in Geometry/HGCalCommonData/python/hgcalV15ParametersInitialization_cfi.py

**Review by perrotta — COMMENTED** [review]
The usage of a safer syntax must be preferred, to prevent possible mistakes.
Ok to defer it to some future cleanup PR

**Comment by perrotta** [comment]
+1

**Pull request merged**